docs/docs/performance.md

```diff
@@ -4,115 +4,122 @@ sidebar_position: 5
+|:--------------------:|:-------------------------------------------------:|:---------------------------------------------------------:|:-------------------------------------------:| 
+|       Average        |                     2647.9 ms                     |                         1221.1 ms                         |                  8149.1 ms                  |
+|        Stdev         |                      14.4 ms                      |                          42.7 ms                          |                 283.575 ms                  |
+| Comparing to snarkjs |                        ~3x                        |           <font color="FFB546">**~6.7x**</font>           |                      -                      |
 
 <iframe width="495" height="252" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbLHpEzT1ybhta5NVQrDQuOLwgGzLGpm2amiWgLRu0l9ZvXMNtNZ-DWIlySL0zO30UWn_nZvkfaQWY/pubchart?oid=333122430&amp;format=interactive"></iframe>
 
-| RSA | [rapidsnark](https://github.com/iden3/rapidsnark) | [ark-works](https://github.com/arkworks-rs/circom-compat) | [snarkjs](https://github.com/iden3/snarkjs) |
-| :--: | :--: | :--: | :--: | 
-| Average | 2908.6 ms | 2324.4 ms | 10304.8 ms |
-| Stdev | 112.9 ms | 67.1 ms | 605.5 ms |
-| Comparing to snarkjs | ~3.5x | <font color="FFB546">**~4.4x**</font> | - |
+|         RSA          | [rapidsnark](https://github.com/iden3/rapidsnark) | [ark-works](https://github.com/arkworks-rs/circom-compat) | [snarkjs](https://github.com/iden3/snarkjs) |
+|:--------------------:|:-------------------------------------------------:|:---------------------------------------------------------:|:-------------------------------------------:| 
+|       Average        |                     2908.6 ms                     |                         2324.4 ms                         |                 10304.8 ms                  |
+|        Stdev         |                     112.9 ms                      |                          67.1 ms                          |                  605.5 ms                   |
+| Comparing to snarkjs |                       ~3.5x                       |           <font color="FFB546">**~4.4x**</font>           |                      -                      |
 
 <iframe width="484" height="266" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTbLHpEzT1ybhta5NVQrDQuOLwgGzLGpm2amiWgLRu0l9ZvXMNtNZ-DWIlySL0zO30UWn_nZvkfaQWY/pubchart?oid=1171109874&amp;format=interactive"></iframe>
 </details>
 
-**Details:** [Spreadsheet of Circom benchmark (iOS)](https://docs.google.com/spreadsheets/d/1MFABmsYSUsWDmhbjleqhBXk7nkYwhu589yK-CHtRkNI/edit?usp=sharing)
+**Details:
+** [Spreadsheet of Circom benchmark (iOS)](https://docs.google.com/spreadsheets/d/1MFABmsYSUsWDmhbjleqhBXk7nkYwhu589yK-CHtRkNI/edit?usp=sharing)
```

Comment:
```suggestion
**Details:** [Spreadsheet of Circom benchmark (iOS)](https://docs.google.com/spreadsheets/d/1MFABmsYSUsWDmhbjleqhBXk7nkYwhu589yK-CHtRkNI/edit?usp=sharing)
```

it shouldn't be a new line or new space after `Details:`

Reply:
Sorry for changes to the file - again this was due to my IDEA formating action on save. Should I rever all the chagnes to the tables, or only fix this one?